### DISCUSSION TITLE
Docs: hardware requirements

### DISCUSSION THREAD

**Discussion by lonix1** [authelia/authelia#6048] [Ideas]
### Description

The docs are comprehensive and of a high quality.

But they are missing details of the app's "minimum hardware requirements": cpu, memory, disk space, etc.

It would be very helpful for new users if that could be added to the "getting started" docs.

### Use Case

New users who need to know what environment to provision.

### Pre-Submission Checklist

- [X] I agree to follow the [Code of Conduct](http://www.authelia.com/code-of-conduct)

- [X] I have checked for related issues and checked the documentation

**Comment by james-d-elliott** [comment]
There are some rough requirements listed here: https://www.authelia.com/

We would welcome a contribution to this section if someone would like: https://www.authelia.com/overview/prologue/introduction/ (maybe a new page in the prologue section called "Hardware Requirements" which the landing page links to).

Edit: Sorry for not getting back to you sooner, I have been rather busy.

**Reply by lonix1** [reply]
Thanks for the reply! Are you referring to this:
> With a compressed container size smaller than 20 megabytes and observed memory usage normally under 30 megabytes, it's one of the most lightweight solutions available.

That really is quite light.

**Reply by james-d-elliott** [reply]
There are a few caveats to that, specifically if you use argon2id with the file user provider the memory allocated to that algorithm will also be used. But we support many other algorithms. As far as CPU usage, I've never (with the exclusion of hashing which is highly tunable and expected to have cost), CPU usage above 0%, I don't think a user has reported this either not that they would without a serious issue maybe.
